packages/video_player/video_player/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.10.12+5
+
+* Locked `video_player_platform_interface` to version `2.1.1` for test to succeed.
```

Comment:
I did not realize this before, but because the test host in the messages in the platform interface were adjusted and the `video_player` **tests** depend on that, CI will fail in the transition period.

So we need to lock the platform interface to the older platform interface version for just this transition period. When part 3/3 is merged, the dependency can be unlocked again.

---

I am sorry, I did not consider this additionally. I think this is the easiest solution. Alternatively, I could simply include the test adjustments in this PR, but I want to include those in part 3/3 instead because that is when the platform interface is logically bumped.

Reply:
Can you please merge this on its own, so the tree becomes green again? That way we can keep merging stuff without ignoring the tooling :P

Reply:
@ditman Sounds great - I will fix it differently in the extra PR however 😉

Reply:
I should have caught this when reviewing, my bad!